samples/app-hello-world/python/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-hello-world-python",
+    "source": "officeDev",
+    "title": "Microsoft Teams python Helloworld Sample",
+    "shortDescription": "Explore a comprehensive Microsoft Teams hello world sample app built with Node.js, demonstrating key features such as tabs, bots, and messaging extensions.",
```

Comment:
Here its mentioned as NodeJS, this is a python sample update description

Reply:
fixed